kiali-server/templates/deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
       - name: {{ . }}
       {{- end }}
       {{- end }}
+      {{- if .Values.deployment.host_aliases }}
+      hostAliases:
+      {{ toYaml .Values.deployment.host_aliases | nindent 8 }}
```

Comment:
missing the `-` prefix and the indent it 2 too many. Because of this, an extraneous newline appears and the indentation is off:

```
      hostAliases:

        - hostnames:
          ...and the rest...
```

With the suggested change, that extraneous newline is removed and indentation is corrected:

```
      hostAliases:
      - hostnames:
        ...and the rest...
```

```suggestion
      {{- toYaml .Values.deployment.host_aliases | nindent 6 }}
```

With that fixed, I can verify the results:

```
kubectl exec -n istio-system $(kubectl get pod -n istio-system -l app=kiali -o name) -- cat /etc/hosts
```

Will show the host entries in /etc/hosts of the kiali server container:

```
# Kubernetes-managed hosts file.
127.0.0.1	localhost
::1	localhost ip6-localhost ip6-loopback

...

# Entries added by HostAliases.
192.168.1.111	foo.local	bar.local
192.168.1.222	foo2.local	bar2.local
```

Reply:
thx for your fix!